Views/AsyncEditor.ValidationSummary.cshtml

```diff
@@ -34,13 +34,3 @@
         </ol>
     }
 </div>
-
-<script type="text/javascript">
-    ; (function ($) {
-        $(function () {
-            $("html, body").animate({
-                scrollTop: $(".js-@BlockName").offset().top
-            }, 500);
-        });
-    })(jQuery);
-</script>
```

Comment:
Ezt miért kellett törölni? Ha jól látom elég sok helyen használva van azért ez a shape.

Reply:
Azt követően, hogy megjelent a modal 500 pixelt elgörgetett így a modal nem maradt a képernyő központjában. Az Event szerkesztői közül egy sem tört el, de igazad van, hogy rengeteg shape használja és biztos van ok arra, hogy belekerült.

Reply:
Szerintem ez nem 500px-t pörget, hanem 500 ms alatt a megadott selector tetejéhez.

Reply:
Correct. "Specifies the speed of the animation."
Nagyon óvatosan ilyen changekkel. Ezt a repositoryt ráadásul nem csak Coliban használjuk.